src/main/java/net/gnomeffinway/depenizen/objects/MythicMobs/MythicMobsMob.java

```diff
@@ -0,0 +1,287 @@
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.LivingEntity;
+
+import java.util.UUID;
+
+public class MythicMobsMob implements dObject, Adjustable {
+
+    public static MythicMobsMob valueOf(String uuid) {
+        return valueOf(uuid, null);
+    }
+
+    @Fetchable("mythicmob")
+    public static MythicMobsMob valueOf(String string, TagContext context) {
+        if (string == null) {
+            return null;
+        }
+        try {
+            string = string.replace("e@", "").replace("mythicmob@", "");
+            UUID uuid = UUID.fromString(string);
+            IMobsAPI api = MythicMobs.inst().getAPI().getMobAPI();
+            if (!api.isMythicMob(uuid)) {
+                return null;
+            }
+            return new MythicMobsMob(api.getMythicMobInstance(dEntity.getEntityForID(uuid)));
+        }
+        catch (Exception e) {
+            return null;
+        }
+    }
+
+    public static boolean matches(String string) {
+        return valueOf(string) != null;
+    }
+
+    public MythicMobsMob(ActiveMob mob) {
+        if (mob != null) {
+            this.mob = mob;
+        }
+        else {
+            dB.echoError("ActiveMob referenced is null!");
+        }
+    }
+
+    public ActiveMob getMob() {
+        return mob;
+    }
+
+    public LivingEntity getLivingEntity() {
+        return mob.getLivingEntity();
+    }
+
+    public Entity getEntity() {
+        return mob.getEntity().getBukkitEntity();
+    }
+
+    private String prefix;
+    ActiveMob mob;
+
+    @Override
+    public dObject setPrefix(String prefix) {
+        this.prefix = prefix;
+        return this;
+    }
+
+    @Override
+    public String getPrefix() {
+        return prefix;
+    }
+
+    @Override
+    public String debug() {
+        return null;
+    }
+
+    @Override
+    public boolean isUnique() {
+        return true;
+    }
+
+    @Override
+    public String getObjectType() {
+        return "MythicMob";
+    }
+
+    @Override
+    public String identify() {
+        return prefix + "='<A>" + identify() + "<G>' ";
+    }
+
+    @Override
+    public String identifySimple() {
+        return identify();
+    }
+
+    @Override
+    public String getAttribute(Attribute attribute) {
+        if (attribute == null) {
+            return null;
+        }
+
+        // <--[tag]
+        // @attribute <mythicmob@mythicmob.level>
+        // @returns Element(Number)
+        // @description
+        // Returns the level of the MythicMob.
+        // @plugin Depenizen, MythicMobs
+        // -->
+        if (attribute.startsWith("level")) {
+            return new Element(mob.getLevel()).getAttribute(attribute.fulfill(1));
+        }
+
+        // <--[tag]
+        // @attribute <mythicmob@mythicmob.players_killed>
+        // @returns Element(Number)
+        // @description
+        // Returns the number of players the MythicMob has killed.
+        // NOTE: Untested and cannot guarantee accuracy of results.
```

Comment:
Don't bother with notes like that. It might discourage but reports that help us actually fix things